packages/bot-web-ui/src/components/dashboard/tutorial-tab/hook/useFilterForTutorials.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { faq_content, guide_content, user_guide_content } from '../config';
+
+export const useFilterTutorialsTab = (search, active_tab, active_tab_tutorials) => {
+    const [guide_tab_content, setGuideContent] = React.useState([...user_guide_content, ...guide_content]);
+    const [faq_tab_content, setFAQContent] = React.useState([...faq_content]);
+    const [tutorials_content, setTutorialTabContent] = React.useState([...guide_tab_content, ...faq_tab_content]);
+    const [filtered_tab_list, setFilteredList] = React.useState([]);
+
+    const removeHTMLTagsFromString = (param = '') => param.replace(/<.*?>/g, '');
+    const filterContentBySearch = (content, search) => removeHTMLTagsFromString(content).toLowerCase().includes(search);
```

Comment:
These could be memoized? So for the same content and search string, it can just return the memorized value?

Reply:
done ✅